### ISSUE TITLE
Can I try improve website's (at least for home page) UI

### ISSUE THREAD

**Issue by Cugatay** [godotengine/godot-website#715]
Hey, I'm a web dev working for a web3 technologies company to create seamless animations and user experiences.

Recently, I reviewed the Godot game engine, and I liked it. Then I looked at its website and while it's explanatory enough, I don't think it's not able to give the experience it should to the Godot community. So, I decided to work on a new design and implement it into code, but not sure if the community and developers are okay with this. I think that changing the whole design language is a bad option, and changing the whole website from home to download page is also not good because I'm sure the developers aren't willing to review a big pull request, and I'm not willing to work on a big request that may be declined neither.

In short, can I try to improve the experience of Godot's website with animations and a great design? For now, I'll work just on the homepage as I said.

**Comment by Calinou** [comment]
> In short, can I try to improve the experience of Godot's website with animations and a great design? For now, I'll work just on the homepage as I said.

We try to avoid animations that automatically play for a few reasons:

- Animations require a lot more CPU/GPU resources than a static website, which can cause battery drain.
- Animations can be distracting from the actual content, and can be unfriendly to people with ADHD.

This is why the homepage uses a random hero image on load as opposed to a carousel, for instance.

**Comment by Cugatay** [comment]
Oh, I get it that's a fair point. Can I try to implement an experience with just some good style and design then?

**Comment by coppolaemilio** [comment]
Hey, I'm closing this one because it is too broad. If you have any proposal for any particular thing you would like to fix feel free to open a PR addressing an issue, or make an issue to discuss a particular feature. Thanks!
